.github/mergify.yml

```diff
@@ -0,0 +1,26 @@
+pull_request_rules:
+  - name: Backport to galactic at reviewers discretion
+    conditions:
+      - base=master
+      - "label=backport-galactic"
+    actions:
+      backport:
+        branches:
+          - galactic
+
+  - name: Backport to foxy at reviewers discretion
+    conditions:
+      - base=master
+      - "label=backport-foxy"
+    actions:
+      backport:
+        branches:
+          - foxy
+
+  - name: Ask to resolve conflict
+    conditions:
+      - conflict
+      - author!=mergify
```

Comment:
In MoveIt2, https://github.com/ros-planning/moveit2/pull/921, they use an additional `[bot]` to avoid tagging itself. Maybe something to consider here.

Reply:
In Moveit2, they added an additional `[bot]` to avoid tagging itself. https://github.com/ros-planning/moveit2/pull/921. Not sure if it is needed here too.

```suggestion
      - author!=mergify[bot]
```

Reply:
It seems that we might not need this for the conflict resolution as it appears here 
https://github.com/ros-planning/moveit2/pull/831. 
MoveIt included it for their "Please use main branch as the base branch".

Reply:
As I understand it now, does the _[bot]_ suffix indicate that this user is a 3rd party application.
If it is really necessary or just used for explicit I couldn't find out using the original documentation.
The configuration of _ros-controls/ros2_control_ for example does not use the [bot]-suffix and it seems to work as well.
So I would recommend for consistency inside of the _ros-controls_ packages to use it without the suffix and if problems arise we can still add the suffix.

Reply:
The only point of having mergify filtered out is that it would ping itself on the PR that would cause a chain of other automated and unnecessary comments.